src/components/SuggestionsList/SuggestionsListItem.tsx

```diff
@@ -22,16 +21,21 @@ export interface SuggestionsListItemProps extends SuggestionItem, NestableBaseCo
 
 const baseClass = 'y-suggestions-list-item';
 const selectedClass = `${baseClass} y-suggestions-list-item--selected`;
+const matchHighlightClass = 'y-suggestions-list-item--match-highlight';
 
 const getHighlightedName = (name: string, search: string) => {
-  return name.split(new RegExp(`(${search})`, 'gi')).map((item: string, index: number) => {
-    const isBold = search.toLowerCase() === item.toLowerCase();
-    return (
-      <Text key={index} bold={isBold}>
-        {item.replace(/\s/g, '\u00A0')}
```

Comment:
are we sure this problem wasn't caused by replacing spaces with non-breaking-space characters, rather than the Text component?

Reply:
checking

Reply:
Here's what that was addressing for `Text`
![screen shot 2018-02-28 at 3 32 00 pm](https://user-images.githubusercontent.com/692240/36819051-94da6cae-1c9c-11e8-9592-8ac705bb4239.png)

Reply:
ah :+1: